unlock-app/src/components/interface/locks/Settings/forms/SendEmailForm.tsx

```diff
@@ -0,0 +1,86 @@
+import { storage } from '~/config/storage'
+import { useMutation, useQuery } from '@tanstack/react-query'
+import { Button, ToggleSwitch } from '@unlock-protocol/ui'
+import { useState } from 'react'
+
+interface SubscriptionFormProps {
+  lockAddress: string
+  network: number
+  isManager: boolean
+  disabled: boolean
+}
+
+export const SendEmailForm = ({
+  lockAddress,
+  network,
+  isManager,
+  disabled,
+}: SubscriptionFormProps) => {
+  const [sendEmail, setSendEmail] = useState(true)
+  const [changed, setChanged] = useState(false)
+
+  const updateRequireEmail = async () => {
+    if (!isManager) return
+    return await storage.saveLockSetting(network, lockAddress, {
+      sendEmail,
+    })
+  }
+
+  const updateSettingsMutation = useMutation(updateRequireEmail, {
+    onSuccess: () => {
+      setChanged(false)
+    },
+  })
+
+  const { isLoading, data: { data: { sendEmail: sendEmailValue } } = {} } =
+    useQuery(
+      [
+        'getLockSettings',
+        lockAddress,
+        network,
+        updateSettingsMutation.isSuccess,
+      ],
+      async () => await storage.getLockSettings(network, lockAddress),
+      {
+        enabled: lockAddress?.length > 0 && !!network && isManager,
+        onSuccess: (res: any) => {
+          setSendEmail(res?.data?.sendEmail ?? true)
+        },
+      }
+    )
+
+  const disabledInput = disabled || isLoading || !isManager
+
+  return (
+    <div className="flex flex-col gap-6">
+      <ToggleSwitch
+        title="Send email"
+        disabled={disabledInput}
+        enabled={sendEmail}
+        description={
+          <span className="mt-2 text-base font-semibold text-black">
+            {!sendEmailValue
+              ? 'Emails are disabled, when a user purchase a key does not receive any email.  '
+              : 'Email are enabled, every time a user purchase a key will receive an email'}
```

Comment:
```suggestion
            {!sendEmailValue
              ? 'Emails are disabled, when a user purchase a key does not receive any email.  '
              : 'Email are enabled, every time a user purchase a key will receive an email'}
```

@julien51  any improvement in the copy here? :)

Reply:
Let me look!